packages/wonder-blocks-core/components/clickable-behavior.test.js

```diff
@@ -47,12 +47,38 @@ describe("ClickableBehavior", () => {
             </ClickableBehavior>,
         );
         expect(button.state("hovered")).toEqual(false);
-        button.simulate("mouseenter");
+        button.simulate("mouseenter", {
+            buttons: 0,
+        });
         expect(button.state("hovered")).toEqual(true);
         button.simulate("mouseleave");
         expect(button.state("hovered")).toEqual(false);
     });
 
+    it("changes only pressed state on mouse enter/leave while dragging", () => {
+        const onClick = jest.fn();
+        const button = shallow(
+            <ClickableBehavior disabled={false} onClick={(e) => onClick(e)}>
+                {(state, handlers) => {
+                    return <button {...handlers}>Label</button>;
+                }}
+            </ClickableBehavior>,
+        );
+        expect(button.state("pressed")).toEqual(false);
+
+        button.simulate("mousedown");
+        button.simulate("dragstart", {preventDefault: jest.fn()});
+        expect(button.state("pressed")).toEqual(true);
+
+        button.simulate("mouseleave");
+        expect(button.state("pressed")).toEqual(false);
+
+        button.simulate("mouseenter", {
+            buttons: 1,
+        });
+        expect(button.state("pressed")).toEqual(true);
```

Comment:
Nice!  I like this test.  I also like how there aren't any changes specific to `dropdown.js`.  I think we may run into some edge cases where a user could drag from a dropdown and release on a button.  I don't know what behavior we want into those scenarios but I think it's sufficiently edge case-y (is that a word?) that we can ignore it for now and follow up with someone the interaction guild later.  We'll probably want to do whatever browsers do by default when using built-in `<select>` and `<button>` components.